examples/tutorials/e2e-development/chat-with-pdf.md

```diff
@@ -282,9 +287,10 @@ We have now explored how to conduct tests and evaluations for prompt flow. Addit
 
 There are several aspects we can experiment with, including but not limited to:
 
-Varying prompts for the rewrite_question and/or QnA steps.
-Adjusting the chunk size or chunk overlap during index building.
-Modifying the context limit.
+* Varying prompts for the rewrite_question and/or QnA steps.
```

Comment:
any special reason to change `-` to `*` here? I used to understand they have same rendering result.

Reply:
I don't think it's resolving correctly if you look at original page "Varying prompts for the rewrite_question and/or QnA steps. Adjusting the chunk size or chunk overlap during index building. Modifying the context limit. These elements can be managed through the "config" object in the flow inputs. If you wish to experiment with the first point (varying prompts), you can add properties to the config object to control this behavior - simply by directing it to different prompt files.".